docs/dev-guides/index.md

```diff
@@ -7,9 +7,9 @@ We're thrilled to have you on board with making the Webaverse a more exciting pl
 
 The Webaverse runs on applications that you can write, package and distribute. These packages are called XRPackages (XRPKs).
 
-We've made it as easy as possible to get started! A few points to start with:
+Follow the guides on the left to get started! A few points to begin with:
 
-- There's no Webaverse SDK! Just make your app using one of the many supported asset types (see below) and quickly package it using our CLI.
+- There's no Webaverse SDK! Just make your app using one of the many supported asset types (see below) and quickly package it using our CLI or our simple drag-and-drop UI at <a href="https://xrpackage.org/inspect.html" target="_blank" rel="noopener noreferrer">xrpackage.org</a>.
```

Comment:
This is technically true that there is no "SDK", I would probably lean into the "Webaverse SDK" term and call that `xrpk`.